### ISSUE TITLE
can't connect to packtpub

### ISSUE THREAD

**Issue by mysiar** [igbt6/Packt-Publishing-Free-Learning#109]
```
[INFO] - Creating session...
[ERROR] - Exception occurred HTTPSConnectionPool(host='www.packtpub.com', port=443): Read timed out. (read timeout=10)
```

**Comment by ad-m** [comment]
It looks like a network disruption on your side. It does not record this kind of problem in your instance. Please note that cron can execute the program, send an e-mail notification, and the mail server will queue the message and deliver it when the network connection returns.

**Comment by mjenczmyk** [comment]
Yeah, it's probably just a random timeout. Can we close this issue @mysiar, or maybe this problem occurs regulary?
